oarepo_communities/ui/communities_components/semantic-ui/js/communities_components/components/CommunityInvitationsModal/CommunityInvitationsModal.jsx

```diff
@@ -0,0 +1,221 @@
+                          validEmailsCount ===
+                            validEmailsCount + invalidEmailsCount
+                            ? "check circle green"
+                            : ""
+                        }
+                      />
+                    }
+                    onChange={(e, { value }) =>
+                      handleChange(value, setFieldValue)
+                    }
+                  />
+                  {invalidEmailsCount > 0 && (
+                    <Label
+                      className="mt-0"
+                      pointing
+                      prompt
+                      content={`${i18next.t(
+                        "Invalid emails"
+                      )}: ${values.emails.invalidEmails.join(", ")}`}
+                    />
+                  )}
+                  <label className="helptext">
+                    {i18next.t(
+                      `Emails shall be provided on separate lines. 
+                    Acceptable formats are johndoe@user.com or Doe John <johndoe@user.com>. 
+                    Note that invitations shall be sent only to the valid email addresses.`
+                    )}
+                  </label>
+                </Form.Field>
+                <Form.Field required className="rel-mt-1">
+                  <FieldLabel label={i18next.t("Role")} />
+                  <List selection>
+                    {usersCanInvite.map((u) => (
+                      <List.Item
+                        key={u.name}
+                        onClick={() => {
+                          if (values.role === u.name) return;
+                          setFieldValue("role", u.name);
+                        }}
+                        active={values.role === u.name}
+                      >
+                        <List.Content>
+                          <List.Header>{u.title}</List.Header>
+                          <List.Description>{u.description}</List.Description>
+                        </List.Content>
+                      </List.Item>
+                    ))}
+                  </List>
+                  <ErrorLabel fieldPath="role" />
+                </Form.Field>
+                <RichInputField
+                  className="rel-mt-1"
+                  label={<FieldLabel label={i18next.t("Message")} />}
+                  fieldPath="message"
+                  optimized
+                  inputValue={() => getIn(values, "message", "")}
+                  initialValue=""
+                />
+              </Form>
+              {successMessage && <Message positive>{successMessage}</Message>}
+              {httpError && <Message negative>{httpError}</Message>}
+            </Modal.Content>
+            <Modal.Actions>
+              <Button
+                onClick={() => {
+                  close();
+                  resetForm();
+                }}
+                floated="left"
+              >
+                <Icon name="remove" /> {i18next.t("Cancel")}
```

Comment:
label should say `Close` (we just close the dialog, not cancel any request in progress)